cloudflare.go

```diff
@@ -405,3 +405,28 @@ type IPsResponse struct {
 	Response
 	Result IPRanges `json:"result"`
 }
+
+type VirtualDNS struct {
+    ID                   string   `json:"id"`
+    Name                 string   `json:"name"`
+    OriginIPs            []string `json:"origin_ips"`
+    VirtualDNSIPs        []string `json:"virtual_dns_ips"`
+    MinimumCacheTTL      uint     `json:"minimum_cache_ttl"`
+    MaximumCacheTTL      uint     `json:"maximum_cache_ttl"`
+    DeprecateAnyRequests bool     `json:"deprecate_any_requests"`
+    ModifiedOn           string   `json:"modified_on"`
+}
+
+type VirtualDNSResponse struct {
```

Comment:
Also, as per #49 it would be great if you moved this and the following struct into `virtualdns.go`.